packages/site/src/ui/change-log.js

```diff
@@ -34,61 +83,17 @@ const ChangeLog = props => {
           margin-left: ${core.layout.spacingXSmall};
         }
       `}</style>
-    </div>
+    </>
   )
 }
-ChangeLog.propTypes = {
-  packageName: PropTypes.string,
+
+ChangeLogLabel.propTypes = {
+  packageName: PropTypes.string.isRequired,
   version: PropTypes.string
 }
 
-const getPackagesApiHost = _ => {
+const getPackagesApiHost = () => {
```

Comment:
no likey the `_`? :)

Reply:
i think it's a linting rule/warning? i know it's a linting rule in typescript so maybe i'm just in the habit

Reply:
i think it's a linting warning because it's an unused var. can't remember